i18n-ai.gemspec

```diff
@@ -32,6 +32,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # Uncomment to register a new dependency of your gem
+  spec.add_development_dependency "rails", "~> 6.1"
```

Comment:
i believe we can even make this simpler by just requiring `railties`. this will also make the `Gemfile.lock` changes to be minimal.

in reference to: https://github.com/svenfuchs/rails-i18n/blob/master/rails-i18n.gemspec